charts/kyverno/values.yaml

```diff
@@ -53,8 +53,8 @@ antiAffinity:
   topologyKey: "kubernetes.io/hostname"
 
 podDisruptionBudget:
-  enabled: false
-  # minAvailable: 1
+#   enabled: false
```

Comment:
Do we still honor this field? Did you mean to remove it here?

Reply:
Yes, I meant to remove enabled here. I'll update this. Thanks!